Airflow retirement public doc updates (#128016)
Azure/azure-content: articles/data-factory/airflow-configurations.md

# Supported Apache Airflow configurations

> [!IMPORTANT]

> > Workflow Orchestration Manager (powered by Apache Airflow) will be permanently retired in Azure Data Factory (ADF) on December 31, 2025.  This Feature is now available in Microsoft Fabric. [Learn more](https://learn.microsoft.com/en-us/fabric/data-factory/apache-airflow-jobs-concepts?source=docs) 

Reviewer: ```suggestion
> Workflow Orchestration Manager (powered by Apache Airflow) will be permanently retired in Azure Data Factory on December 31, 2025. The feature is now available in Microsoft Fabric. [Learn more](https://learn.microsoft.com/fabric/data-factory/apache-airflow-jobs-concepts?source=docs)
>
```

> > We recommend migrating all Workflow Orchestration Manager (Apache Airflow in ADF) workloads to Data Workflows (Apache Airflow in Microsoft Fabric) to take advantage of the expanded capabilities before December 31, 2025.

Reviewer: ```suggestion
> We recommend that you migrate all Workflow Orchestration Manager (Apache Airflow in Azure Data Factory) workloads to Data Workflows (Apache Airflow in Microsoft Fabric) to benefit from expanded capabilities before December 31, 2025.
```

> >For additional guidance or support during your migration to Fabric Airflow, please contact Microsoft Support team.

Reviewer: ```suggestion
>
> For more information or for support during your migration to Apache Airflow in Microsoft Fabric, contact Microsoft Support.
```
